lib/jsdom/living/nodes/RadioNodeList-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const NodeListImpl = require("./NodeList-impl").implementation;
+
+class RadioNodeListImpl extends NodeListImpl {
+  get value() {
+    this._update();
+    const element = this._list.find(e => e._localName === "input" && e.type === "radio" && e.checked);
+    if (element && element.value !== "") {
```

Comment:
Let's follow the spec more exactly here. It has a step that says "if element is null, return the empty string". Please translate that, in this case becoming `if (element === undefined) { return ""; }`.

Then it says, "If element is an element with no [value](https://html.spec.whatwg.org/#attr-input-value) attribute, return the string "on"." This is not very well-written spec text, but again, let's try to translate it. In this case it'd be `if (!("value" in element)) { return "on"; }`.

And then there's a final step... that one should be clear.